lib/generators/templates/setup/dev_prime.rb

```diff
@@ -0,0 +1,12 @@
+if Rails.env.development? || Rails.env.test?
+  require "factory_bot" if Bundler.rubygems.find_name("factory_bot_rails").any?
+
+  namespace :dev do
+    desc "Sample data for local development environment"
+    task prime: "db:setup" do
```

Comment:
This seemed to be wiping the database, even when `WIPE_DATABASE` was not set.

Reply:
It seems `db:setup` is wipes everything. d28306358f097b7b73f4cff56bd6a5e616f3dcdb

Reply:
d283063